doc-botonic/docs/concepts/18.webchat.md

```diff
@@ -30,6 +30,156 @@ Botonic supports the following browsers:
+
+To add polyfills:
+
+1. Open the `webpack.config.js` file.
+
+2. Add all the polyfills that are currently used in the entry property within the `botonicWebchatConfig`. The first 2 polyfills are mandatory, as well as the 2 that follows:
+
+```javascript
+entry: [
+  'core-js/stable/',
+  'regenerator-runtime/runtime',
+  '@webcomponents/webcomponentsjs',
+  'element-scroll-polyfill',
+  'unorm',
+  path.resolve('src', 'webchat-entry.ts'),
+],
+```
+
+#### Configure the Babel loader
+
+When configuring the Babel loader to transpile code to ES5 compliance, some options must be added/configured.
+
+1. Set `sourceType` to 'unambiguous' (cf. issue: https://github.com/babel/babel/issues/8900 )
+
+2. In presets, the definition order is important. Therefore, `babel/env` preset needs to be set before typescript to avoid losing constructor auto-assign: (https://github.com/babel/babel/issues/8752#issuecomment-486541662)
+
+3. Add plugins to transform some new functionalities from ES6 to ES5 compliant. For example: the arrow functions.
+
+```javascript
+options: {
+  sourceType: 'unambiguous',
+  cacheDirectory: true,
+  presets: [
+    '@babel/react',
+    [
+      '@babel/env',
+      {
+        targets: {
+          ie: '11',
+        },
+        useBuiltIns: 'entry',
+        corejs: 3,
+        modules: false, // Needed for tree shaking to work.
+      },
+    ],
+    '@babel/typescript',
+  ],
+  plugins: [
+    require('@babel/plugin-proposal-object-rest-spread'),
+    require('@babel/plugin-proposal-class-properties'),
+    //require('babel-plugin-add-module-exports'),
+    require('@babel/plugin-transform-runtime'),
+    require('@babel/plugin-transform-object-assign'),
+    require('@babel/plugin-transform-arrow-functions'),
+    require('@babel/plugin-transform-classes'),
+  ],
+}
+```
+
+#### Load the bot and correct the dependencies
+
+You can now load the bot in IE11. If an error still occurs despite the correct configuration in `webpack.config.js`, you can solve them by finding the error and the dependency that produces the error.
+
+To do so:
+
+1. Open the developer tool (F12).
+2. Click on the error messages line to display the exact part of the code where the error occurs.
+3. Try to find some parts of the code that are easy to search.
+4. Search for that part in the `node_modules` folder to see which dependency contains that piece of code.
+5. Once the dependency that produces the error is found, add it to babel in order to transpile its code to ES5.
+6. Add the name of the dependency in the exclude option (regexp) to include it.
+
+Example with babel transpiling the dependency "axios":
+
+Before: `exclude: /node_modules[\\/\\\\](?!(styled-components|@emotion|@rehooks|@botonic\\/(core|react|plugin-google-analytics))[\\/\\\\])/,`
+
+After: `exclude: /node_modules[\\/\\\\](?!(**axios**|styled-components|@emotion|@rehooks|@botonic\\/(core|react|plugin-google-analytics))[\\/\\\\])/,`
```

Comment:
I can't see the "**axios**" text in bold when seeing the [markdown rendered](https://github.com/hubtype/botonic/blob/61d19ba08206dbe5c29ac4468467bbfe6bf99d79/doc-botonic/docs/concepts/18.webchat.md). Will it be transformed into bold correctly? If it won't, maybe it will be better to leave it without bold format and remove the "*".

Also there are too backslashes and I think it would be better to show it as a list, I think the final code should look like this:

- Before: `exclude: /node_modules[\/\\](?!(styled-components|@emotion|@rehooks|@botonic\/(core|react|plugin-google-analytics))[\/\\])/,`
- After: `exclude: /node_modules[\/\\](?!(axios|styled-components|@emotion|@rehooks|@botonic\/(core|react|plugin-google-analytics))[\/\\])/,`
```suggestion
- Before: `exclude: /node_modules[\/\\](?!(styled-components|@emotion|@rehooks|@botonic\/(core|react|plugin-google-analytics))[\/\\])/,`
- After: `exclude: /node_modules[\/\\](?!(axios|styled-components|@emotion|@rehooks|@botonic\/(core|react|plugin-google-analytics))[\/\\])/,`
```

Reply:
Thanks, I've just modified it and committed the changes.